2019/01/06 03:15:21 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'testdrive.tbl_migration' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `tbl_migration`.
2019/01/06 04:24:01 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'testdrive.project' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `project`.
in /Applications/MAMP/htdocs/trackstar/index.php (13)
2019/01/06 04:24:36 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'testdrive.project' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `project`.
in /Applications/MAMP/htdocs/trackstar/index.php (13)
2019/01/06 04:36:14 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '' for column 'create_time' at row 1. The SQL statement executed was: INSERT INTO `tbl_project` (`name`, `description`, `create_time`, `create_user_id`, `update_time`, `update_user_id`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5).
in /Applications/MAMP/htdocs/trackstar/protected/controllers/ProjectController.php (73)
in /Applications/MAMP/htdocs/trackstar/index.php (13)
2019/01/06 04:36:14 [error] [exception.CDbException] CDbException: CDbCommand failed to execute the SQL statement: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '' for column 'create_time' at row 1. The SQL statement executed was: INSERT INTO `tbl_project` (`name`, `description`, `create_time`, `create_user_id`, `update_time`, `update_user_id`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5) in /Applications/MAMP/htdocs/yii-1.1.19/framework/db/CDbCommand.php:358
Stack trace:
#0 /Applications/MAMP/htdocs/yii-1.1.19/framework/db/ar/CActiveRecord.php(1083): CDbCommand->execute()
#1 /Applications/MAMP/htdocs/yii-1.1.19/framework/db/ar/CActiveRecord.php(811): CActiveRecord->insert(NULL)
#2 /Applications/MAMP/htdocs/trackstar/protected/controllers/ProjectController.php(73): CActiveRecord->save()
#3 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/actions/CInlineAction.php(49): ProjectController->actionCreate()
#4 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/filters/CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/filters/CFilter.php(40): CFilterChain->run()
#7 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/CController.php(1148): CFilter->filter(Object(CFilterChain))
#8 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/filters/CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/CController.php(291): CFilterChain->run()
#11 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/CWebApplication.php(282): CController->run('create')
#13 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/CWebApplication.php(141): CWebApplication->runController('project/create')
#14 /Applications/MAMP/htdocs/yii-1.1.19/framework/base/CApplication.php(185): CWebApplication->processRequest()
#15 /Applications/MAMP/htdocs/trackstar/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/trackstar/index.php?r=project/create
HTTP_REFERER=http://localhost/trackstar/index.php?r=project/create
---
2019/01/06 04:36:39 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '' for column 'create_time' at row 1. The SQL statement executed was: INSERT INTO `tbl_project` (`name`, `description`, `create_time`, `create_user_id`, `update_time`, `update_user_id`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5).
in /Applications/MAMP/htdocs/trackstar/protected/controllers/ProjectController.php (73)
in /Applications/MAMP/htdocs/trackstar/index.php (13)
2019/01/06 04:36:39 [error] [exception.CDbException] CDbException: CDbCommand failed to execute the SQL statement: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '' for column 'create_time' at row 1. The SQL statement executed was: INSERT INTO `tbl_project` (`name`, `description`, `create_time`, `create_user_id`, `update_time`, `update_user_id`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5) in /Applications/MAMP/htdocs/yii-1.1.19/framework/db/CDbCommand.php:358
Stack trace:
#0 /Applications/MAMP/htdocs/yii-1.1.19/framework/db/ar/CActiveRecord.php(1083): CDbCommand->execute()
#1 /Applications/MAMP/htdocs/yii-1.1.19/framework/db/ar/CActiveRecord.php(811): CActiveRecord->insert(NULL)
#2 /Applications/MAMP/htdocs/trackstar/protected/controllers/ProjectController.php(73): CActiveRecord->save()
#3 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/actions/CInlineAction.php(49): ProjectController->actionCreate()
#4 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/filters/CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/filters/CFilter.php(40): CFilterChain->run()
#7 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/CController.php(1148): CFilter->filter(Object(CFilterChain))
#8 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/filters/CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/CController.php(291): CFilterChain->run()
#11 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/CWebApplication.php(282): CController->run('create')
#13 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/CWebApplication.php(141): CWebApplication->runController('project/create')
#14 /Applications/MAMP/htdocs/yii-1.1.19/framework/base/CApplication.php(185): CWebApplication->processRequest()
#15 /Applications/MAMP/htdocs/trackstar/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/trackstar/index.php?r=project/create
HTTP_REFERER=http://localhost/trackstar/index.php?r=project/create
---
2019/01/06 04:41:26 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in /Applications/MAMP/htdocs/yii-1.1.19/framework/web/auth/CAccessControlFilter.php:183
Stack trace:
#0 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/auth/CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/filters/CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/filters/CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/CController.php(291): CFilterChain->run()
#6 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/CWebApplication.php(282): CController->run('admin')
#8 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/CWebApplication.php(141): CWebApplication->runController('project/admin')
#9 /Applications/MAMP/htdocs/yii-1.1.19/framework/base/CApplication.php(185): CWebApplication->processRequest()
#10 /Applications/MAMP/htdocs/trackstar/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/trackstar/index.php?r=project/admin
HTTP_REFERER=http://localhost/trackstar/index.php?r=project/index
---
2019/01/06 04:46:03 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in /Applications/MAMP/htdocs/yii-1.1.19/framework/web/auth/CAccessControlFilter.php:183
Stack trace:
#0 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/auth/CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/filters/CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/filters/CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/CController.php(291): CFilterChain->run()
#6 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/CWebApplication.php(282): CController->run('delete')
#8 /Applications/MAMP/htdocs/yii-1.1.19/framework/web/CWebApplication.php(141): CWebApplication->runController('project/delete')
#9 /Applications/MAMP/htdocs/yii-1.1.19/framework/base/CApplication.php(185): CWebApplication->processRequest()
#10 /Applications/MAMP/htdocs/trackstar/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/trackstar/index.php?r=project/delete&id=2
HTTP_REFERER=http://localhost/trackstar/index.php?r=project/view&id=2
---
